client-js/main/client/actor-request-factory.js

```diff
@@ -42,11 +42,21 @@ import {Type, TypedMessage, isProtobufMessage} from './typed-message';
 import {AnyPacker} from './any-packer';
 import {FieldPaths} from './field-paths';
 
+// TODO:2019-06-07:yegor.udovchenko: Cover `Filters` class withunit tests
+// https://github.com/SpineEventEngine/web/issues/100
 /**
  * A factory for `Filter` and `CompositeFilter` instances.
  */
 export class Filters {
 
+  /**
+   * @typedef {string | number | boolean | TypedMessage<T> | <T extends Message>} FieldValue
```

Comment:
We also have time-based values. What if a JS `Date`-typed value is passed? We need to convert it into a `Timestamp` message.